newm-server/src/main/kotlin/io/projectnewm/server/ext/JsonPrimitiveExt.kt

```diff
@@ -0,0 +1,9 @@
+package io.projectnewm.server.ext
+
+import kotlinx.serialization.json.JsonPrimitive
+import kotlinx.serialization.json.booleanOrNull
+import kotlinx.serialization.json.double
+import kotlinx.serialization.json.longOrNull
+
+val JsonPrimitive.value: Any
+    get() = if (isString) content else booleanOrNull ?: longOrNull ?: double
```

Comment:
should the last one be `doubleOrNull`? or are these the only primitives it supports?

Reply:
Yeah, those are the only JSON primitives possible